packages/swarm-components/src/TogglePill.jsx

```diff
@@ -2,53 +2,52 @@
 import * as React from 'react';
 
 export type Props = {
-  /**
-   * Indicates whether the toggle is selected
-   */
-  checked?: boolean,
-  /**
-   * Indicates whether the toggle is disabled
-   */
-  disabled?: boolean,
-  /**
-   * Text label of the content
-   */
-  label?: string,
-  /**
-   * Grow select to 100% width (full width)
-   */
-  grow?: boolean,
-  /**
-   * Text label of the content, applied if label is not set
-   */
-  children?: React.Node
+	/**
+	 * Indicates whether the toggle is selected
+	 */
+	checked?: boolean,
+	/**
+	 * Indicates whether the toggle is disabled
+	 */
+	disabled?: boolean,
+	/**
+	 * Text label of the content
+	 */
+	label?: string,
+	/**
+	 * Grow select to 100% width (full width)
+	 */
+	grow?: boolean,
+	/**
+	 * Text label of the content, applied if label is not set
+	 */
+	reset?: boolean,
+	/**
+	 * Strips all styles from the toggle pill
+	 */
+	children?: React.Node,
 };
 
 const TogglePill = (props: Props): React.Element<'button'> => {
-  const {
-    checked,
-    disabled,
-    label,
-    children,
-    grow,
-    ...rest
-  } = props;
+	const { checked, disabled, label, children, grow, reset, ...rest } = props;
```

Comment:
It predates this PR, but let's rename `rest` to `other`. I believer all the other components use `other`, but also `reset` and `rest` are so similarly spelled that it might just be better to avoid confusion/typos.